family.pl

```diff
@@ -176,4 +216,16 @@
 % Impress me.
 %
 
+
+
+
+auntCheck(X,Y) :- parent(A,X),parent(A,Y),female(X);
+		  parent(Y,Z),auntCheck(Z,X).
+/* if A is parent of X and A is Y's parent and X is famale and Y is parent of Z
+then Z is aunt of X. */
```

Comment:
This is close, but you are actually saying.  X is Y's aunt if:

1. A is parent to both X and Y and X is female (X is a sister of Y), or
2. Y is the parent of Z for some Z that satisfies auntCheck(Z,X).

This rule does not need to be recursive.  You can do it all in a predicate with four goals:

```prolog
auntCheck(X,Y) :- parent(A,X),parent(A,Z),female(X),parent(Z,Y).
```